instrumentation/jdbc/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jdbc/test/JdbcInstrumentationTest.java

```diff
@@ -0,0 +1,1365 @@
+            null,
+            "SELECT 3",
+            "SELECT ?",
+            "SELECT " + dbNameLower,
+            "h2:mem:",
+            null),
+        Arguments.of(
+            "derby",
+            cpDatasources.get("tomcat").get("derby").getConnection(),
+            "APP",
+            "SELECT 3 FROM SYSIBM.SYSDUMMY1",
+            "SELECT ? FROM SYSIBM.SYSDUMMY1",
+            "SELECT SYSIBM.SYSDUMMY1",
+            "derby:memory:",
+            "SYSIBM.SYSDUMMY1"),
+        Arguments.of(
+            "hsqldb",
+            cpDatasources.get("tomcat").get("hsqldb").getConnection(),
+            "SA",
+            "SELECT 3 FROM INFORMATION_SCHEMA.SYSTEM_USERS",
+            "SELECT ? FROM INFORMATION_SCHEMA.SYSTEM_USERS",
+            "SELECT INFORMATION_SCHEMA.SYSTEM_USERS",
+            "hsqldb:mem:",
+            "INFORMATION_SCHEMA.SYSTEM_USERS"),
+        Arguments.of(
+            "h2",
+            cpDatasources.get("hikari").get("h2").getConnection(),
+            null,
+            "SELECT 3",
+            "SELECT ?",
+            "SELECT " + dbNameLower,
+            "h2:mem:",
+            null),
+        Arguments.of(
+            "derby",
+            cpDatasources.get("hikari").get("derby").getConnection(),
+            "APP",
+            "SELECT 3 FROM SYSIBM.SYSDUMMY1",
+            "SELECT ? FROM SYSIBM.SYSDUMMY1",
+            "SELECT SYSIBM.SYSDUMMY1",
+            "derby:memory:",
+            "SYSIBM.SYSDUMMY1"),
+        Arguments.of(
+            "hsqldb",
+            cpDatasources.get("hikari").get("hsqldb").getConnection(),
+            "SA",
+            "SELECT 3 FROM INFORMATION_SCHEMA.SYSTEM_USERS",
+            "SELECT ? FROM INFORMATION_SCHEMA.SYSTEM_USERS",
+            "SELECT INFORMATION_SCHEMA.SYSTEM_USERS",
+            "hsqldb:mem:",
+            "INFORMATION_SCHEMA.SYSTEM_USERS"),
+        Arguments.of(
+            "h2",
+            cpDatasources.get("c3p0").get("h2").getConnection(),
+            null,
+            "SELECT 3",
+            "SELECT ?",
+            "SELECT " + dbNameLower,
+            "h2:mem:",
+            null),
+        Arguments.of(
+            "derby",
+            cpDatasources.get("c3p0").get("derby").getConnection(),
+            "APP",
+            "SELECT 3 FROM SYSIBM.SYSDUMMY1",
+            "SELECT ? FROM SYSIBM.SYSDUMMY1",
+            "SELECT SYSIBM.SYSDUMMY1",
+            "derby:memory:",
+            "SYSIBM.SYSDUMMY1"),
+        Arguments.of(
+            "hsqldb",
+            cpDatasources.get("c3p0").get("hsqldb").getConnection(),
+            "SA",
+            "SELECT 3 FROM INFORMATION_SCHEMA.SYSTEM_USERS",
+            "SELECT ? FROM INFORMATION_SCHEMA.SYSTEM_USERS",
+            "SELECT INFORMATION_SCHEMA.SYSTEM_USERS",
+            "hsqldb:mem:",
+            "INFORMATION_SCHEMA.SYSTEM_USERS"));
+  }
+
+  @DisplayName(
+      "basic statement with #connection.getClass().getCanonicalName() on #system generates spans")
```

Comment:
I'm not sure about the value of this. Spock is able to replace `#connection.getClass().getCanonicalName()` and `#system` with actual parameter values so you'd have a test named `prepared statement execute on h2 with org.h2.jdbc.JdbcConnection generates a span` with junit this does not happen

Reply:
junit can not do the same ability，like getCannicalname。

Junit is a tree struct， DisplayName for the method， parametertest have each name。

I do not found  the usage in other tests in the projects。

Reply:
I don't think you understood. When you run the groovy tests in intellij then the main test is named `basic statement with #connection.getClass().getCanonicalName() on #system generates spans` when you expand it then you see the test with parameters. They are named `basic statement with org.h2.jdbc.JdbcConnection on h2 generates spans`. When you run the junit test the main test name is what you set in display name and when you expand it you'll just see the parameters. That is why I believe that using the `@DisplayName` they way you do does not make sense. There is no magic interpolation like in spock. You might either not add `@DisplayName` at all or change it to `@DisplayName("basic statement generates spans")`

Reply:
i will remove @DisplayName

Reply:
done